JASP-Desktop/analysis/analysis.h

```diff
@@ -170,6 +170,7 @@ class Analysis : public QObject
 	void				imageEditedSignal(		Analysis * analysis);
 	void				resultsChangedSignal(	Analysis * analysis);
 	void				userDataChangedSignal(	Analysis * analysis);
+	void				somethingModified(); //Possibly not emitted everywhere. But it should be ^^ (well, eventually, but take into account loading stuff etc, thats why it wasnt added everywhere it should be in the first place)
```

Comment:
I don't think that a new signal should be added.
I am not for adding a signal for each purpose. The name of a signal should only described what has changed, and it should be connected to all needed slots, eg. analysis.titleChanged -> analyses.somethingChanged. It has the advantage that if titleChanged signal is emitted somewhere else, then you don't have to remember that you have to call also the analysis somethingChanged signal.

Reply:
I guess that makes sense.

Reply:
Ok, so thats fixed.